### PR TITLE
Removes rev from rotation

### PR DESCRIPTION
**Pull request by monster860**
It is literally "lel murderbone heads" and is not fun in any way. Literally "You picked head? Go fuck yourself you're a ghost now" for literally no reason. I'm having a perfectly good round and suddenly out of the blue like 5 people walk up to me and murderize me for no reason.

:cl: monster860
del: remove rev from rotation
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
who hurt you




good pr tho

**Comment by Time-Green** [comment]
Doesn't work, it needs to be done from the host. 
But yes, I agree, rev is shit.
Also remove gang while we're at it

**Comment by matskuman5** [comment]
Removes nukeops from rotation

It is literally "lel murderbone captain" and is not fun in any way. Literally "You picked captain? Go fuck yourself you're a ghost now" for literally no reason. I'm having a perfectly good round and suddenly out of the blue like 5 redsuits walk up to me and murderize me for no reason.

Removes blob from rotation.

It is literally "lel murderbone crew" and is not fun in any way. Literally "You picked crew? Go fuck yourself you're a ghost now" for literally no reason. I'm having a perfectly good round and suddenly out of the blue like 5 spores walk up to me and murderize me for no reason.

Removes cult from rotation.

It is literally "lel murderbone chaplain" and is not fun in any way. Literally "You picked chaplain? Go fuck yourself you're a ghost now" for literally no reason. I'm having a perfectly good round and suddenly out of the blue like 5 people with blank papers walk up to me and murderize me for no reason.

:rage: 

Honestly rev seems fine to me. If you're half decent and you notice that people are being flashed, you arm yourself and just genocide the revs.

**Comment by ShadowDeath6** [comment]
to be fair, rev is bad in the same way that gang is bad, meaning "us vs them" and everyone besides loyalty implants or revs can go fuck themselves which isn't fun for anyone besides the 3 revs using bombs or chems or whatever autism they can use to valid sec and heads and the round is either 5 minutes because revs spread like the aids that they are or heads turtle and wait for an hour and a half while sec massacres anyone that doesn't comply for their mindslave implant

**Comment by monster860** [comment]
The problem with rev is that converting is insanely easy, so without anyone at all ever knowing the numbers grow to 15 or 20 and then they all just murderize the heads.

If anything make the CE, CMO, and RD exempt from being murderboned because they don't get any guns at all.

**Comment by monster860** [comment]
@matskuman5 All the other examples, there is at least some warning. With rev, unless someone fucks up, there is 0 warning that there is any rev at all until it's too late.

**Comment by ghost** [comment]
@monster860 Wrong, actually.

Rev was changed a while back to give conversion a significant cooldown through use of a 'suspicious device' so you can convert one or two guys every few minutes as opposed to 10 guys in 1 minute. Unless you convert more flashes which bypasses the point I guess.

Also this is a .txt file. Editing it does nothing. Ask a council member to change it.

**Comment by Jc1469** [comment]
long ago rev and cult were both disabled, everything changed when brains attacked.
both were enabled, people started having... fun..?  But all good must come to an end.

The coder nation mustered all of their strength to make the most crap thing ever imagined, GANG.  Through the use of GANG, every gamemode became more and more shit, the server became more and more shit, and eventually it got removed.



Tl;dr remove gang not rev gang sucks and we did actually have both gang and rev disabled permanently once upon a time

**Comment by ShadowDeath6** [comment]
The difference between gang and rev is that in gang you die regardless of what you do and in rev you only die half of the time
The similiarity is that they're both objectively awful modes

**Comment by drderp3635** [comment]
*stubs toe on rev
"What the fuck is this, banned"

**Comment by BOBsHERE** [comment]
This is a joke pr right? All of you people commenting seriously considering removing rev are kidding right?

**Comment by Time-Green** [comment]
I was joking about the rev part, gang however, can fucking burn

**Comment by Mordokh** [comment]
Rev is great, makes the crew work for a common goal, why would you disable it?

**Comment by Stealthkibbler** [comment]
lets just remove any gamemode and bring back traitorchan and only let traitorchan to trigger.

**Comment by drderp3635** [comment]
How long till this PR get the meme zone tag?

**Comment by Jc1469** [comment]
how long till it gets merged more like.  just merge it already cruix.    and then laugh because merging it did nothing because it can't change configs.

**Comment by Cruix** [comment]
For the record even though this won't affect the host, this is how you should go about trying to change the configs since people will actually see it.

Also improve don't remove.
